### DISCUSSION TITLE
How to change date picker title and controls' language?

### DISCUSSION THREAD

**Discussion by Luxney81** [henninghall/react-native-date-picker#550] [Q&A] [answered]
How do I change controls' language, if I chose, for example, "he" locale.

![Screen Shot 2022-07-20 at 3 43 27 PM](https://user-images.githubusercontent.com/25150475/179984957-9456e6d8-b3cf-43c1-be07-1218d6039db5.png)
![Screen Shot 2022-07-20 at 3 43 33 PM](https://user-images.githubusercontent.com/25150475/179984962-f9f85aec-a0fa-429c-ade0-65da6bd7b526.png)

**Comment by henninghall** [comment, marked as answer]
There are props for this, have a look at the docs 🙂
